docs/infrastructure/security/overview.md

```diff
@@ -0,0 +1,62 @@
+---
+id: security-overview
+title: Mia-Platform Console Security
+sidebar_label: Security
+---
+
+To help prevent [supply-chain attacks], some Mia-Platform released artifacts are cryptographically signed so you can
+be sure to have downloaded are the ones built and distributed by Mia-Platform.
+
+You can verify the signing in every momento to be sure that there was no tampering of the released artifact that you
+are about to install.
+
+The public certificates that are required for the verification are available on this website and may depend on the
+artifact you need to verify. At this time of writing all the artifacts are signed using the same underling key.
+
+## Container Images
+
+Here you can find all the artifacts and the starting version when they are being signed and verifiable with `cosign`:
+
+| Container Image | Starting Tag |
+| --- | --- |
+| nexus.mia-platform.eu/console/backend | v10.0.0 |
```

Comment:
Before merging this PR, at least one real service must be inserted in the table.

Reply:
```suggestion
| nexus.mia-platform.eu/rond-authz/rond | v1.12.9 |
```